src/assets/cta.scss

```diff
@@ -1,72 +1,76 @@
 @import "./main";
 
-.ra21-button {
-  background-color: var(--primary-blue);
-  border-radius: 5px;
-  color: var(--warm-grey);
-  cursor: pointer;
-  display: flex;
-  margin-bottom: 3px;
-  padding: 9px;
-  width: 100%;
-  text-decoration: none;
+body {
```

Comment:
This centers everything in the CTA element since everything is centered anyways. This specifically takes care of the <a> element which can only be centered via its parent--which in our case is <body>.